presto-native-tests/src/test/java/com/facebook/presto/nativetests/AbstractTestAggregationsNative.java

```diff
@@ -38,7 +38,7 @@ public void init(String storageFormat, boolean sidecarEnabled)
             charTypeUnsupportedError = ".*Unknown type: char.*";
             timeTypeUnsupportedError = ".*Unknown type: time.*";
             approxDistinctUnsupportedSignatureError = ".*Unexpected parameters \\(timestamp with time zone.*\\) for function.*";
-            mergeAggFunctionUnsupportedError = ".*Unexpected parameters \\(qdigest.* for function native.default.merge.*";
+            mergeAggFunctionUnsupportedError = ".*Unexpected parameters.*";
```

Comment:
@amitkdutta : Can you share what is the full error message now ?

Reply:
Could we make the error message specific to the tdigest type, for which the merge function is not supported yet?
Also, could you change the error message for sidecar disabled as well?